shopify/customized-bundle-template/api/actions/updateBundleInShopify.ts

```diff
@@ -47,7 +41,9 @@ export async function run({ params, logger, api, connections }) {
   }
 
   if (variantChanges?.length) {
-    const { id: variantId, ...variantData } = variant;
+    if (!bundle?.variant?.id) throw new Error("Bundle variant ID not found");
+
+    const { id: variantId, ...variantData } = bundle?.variant;
 
     // Update the product variant with the new data
     const productVariantUpdateResponse = await shopify.graphql(
```

Comment:
This API is no longer supported in 2024-10, the template will not work 
https://shopify.dev/docs/api/admin-graphql/2024-10/mutations/productVariantUpdate

Reply:
We can now create and update bundles without an issue